packages/realm/src/app-services/User.ts

```diff
@@ -70,6 +70,9 @@ export class User<
   /** @internal */
   public internal: binding.SyncUser;
 
+  // cached version of profile
+  private _profile: UserProfileDataType | undefined;
```

Comment:
we talked about not using underscore for private fields but I'm not sure how we actually want it so putting it out there like this for now

Reply:
I would name it `cachedProfile` and remove the `_`.